### DISCUSSION TITLE
Changing Mode doesn't change corresponding Windows Power Plan

### DISCUSSION THREAD

**Discussion by slikvik55** [seerge/g-helper#2667] [Q&A] [answered]
Is this correct:

![image](https://github.com/seerge/g-helper/assets/17570255/93ee07a8-b2ea-40b3-9077-4b1797ef1911)

**Comment by seerge** [comment]
@slikvik55 absolutely. Please [check readme ](https://github.com/seerge/g-helper?tab=readme-ov-file#rocket-performance-modes) :)

**Reply by slikvik55** [reply]
Hi @seerge I always appreciate your fast support of this! 

I read the readme before but I'm still confused:
![image](https://github.com/seerge/g-helper/assets/17570255/f21ab8d5-4b9e-4985-af98-5873590ca861)

I set **Turbo** in GHelper and it stays as **Balanced** in Windows? (In fact the Windows Power Mode doesn't change no matter what mode I set in GHelper). 

If that IS correct - that it always stays on Balanced - that isn't what I see from reading the readme.  I read it as "_each BIOS mode is paired with **matching** Windows Power Mode_" reads like it will change the Windows mode to match.

**Reply by seerge** [reply, marked as answer]
@slikvik55 you are mixing **power modes** and **power plans**.

In modern windows "power modes" have replaced legacy power plans (practically). They can be set under  Windows ``Settings-> Power & Battery`` and G-Helper sets them.
![image](https://github.com/seerge/g-helper/assets/5920850/f03c8e4a-f5ae-47bb-8289-d5a62aeb82dc)

Power plan stays "balanced" and this is as it should be. 

If you really want to override power plan with something custom or your own [you can do that](https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode), but there is not much sense it it :)

**Reply by slikvik55** [reply]
Thank you!!! (I guess I'm just being very old and used to navigating to the same Power Mode area since Windows Vista!! 😂 )
